src/routes/recovery/guards/disable-recovery-alerts.guard.ts

```diff
@@ -67,5 +69,17 @@ export class DisableRecoveryAlertsGuard implements CanActivate {
       this.loggingService.debug(e);
       return false;
     }
+
+    try {
+      const { safes } = await this.safeRepository.getSafesByModule({
+        chainId,
+        moduleAddress,
+      });
+
+      return safes.some((safe) => getAddress(safe) === getAddress(safeAddress));
+    } catch (e) {
+      this.loggingService.debug(e);
+      return false;
+    }
```

Comment:
We can merge the try/catch block with the one above. Alternatively you can extract this part to a new `Guard` in favour of composability (e.g.: `SafeHasModuleGuard`).

However for the context of this PR and since we only perform this check here I would say that we can merge the try/catch block first.

Reply:
Merged in c3d6cff8a5ccbc4f53a9e597513c0d73cc46b7f8.